framework/resources/Datasources/hive_storage/skipHeaderFooter/DRILL_5941_prepare_data.sh

```diff
@@ -25,12 +25,9 @@ copy_to_dfs () {
     then
         hadoop fs -mkdir -p $destination
     fi
-
-    hadoop fs -test -f $destination/$file_name
-    if [ ! $? -eq 0 ]
-    then
-	    hadoop fs -put $file $destination/
-    fi
+    
+    # Replace the csv every time
+    hadoop fs -put -f $file $destination/
```

Comment:
Doesn't this increase the run time always? If it's a small increase, it is probably ok.  I'm wondering if there are better ways to check for potential corruption, like an MD5 checksum on a test dataset, though I'm not sure there is a hadoop command to do the same.

Reply:
Yeah, that's true. Here since it's a data gen scripts which is doing the copy, we anyway delete and re-create data - this is across scripts. This was wasn't doing that and hence the fix.

As part of test framework data copy functionality we could think of the checksum approach. 

Although I think overwriting is fine as these are done only on a nightly basis - so increase in total runtime won't impact most runs.